packages/backend-core/src/environment.ts

```diff
@@ -66,6 +66,7 @@ const env: any = {
 for (let [key, value] of Object.entries(env)) {
   // handle the edge case of "0" to disable an environment variable
   if (value === "0") {
+    // @ts-ignore
     env[key] = 0
   }
```

Comment:
Nice one, this is a better fix that the any I had put in which was a real bother